rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2021-05-30
```

Comment:
Nice, but I'm worried this will cause confusion, but I guess this is better than what we have.

Reply:
rust.nix can actually handle this I think, but you still need to update the hash